test/stressGraphics.cxx

```diff
@@ -81,390 +81,56 @@
-      std::cout << "*  Starting complex drawing and TPad - S T R E S S                   *" <<std::endl;
-      std::cout << "**********************************************************************" <<std::endl;
-   }
-   ntuple1       ();
-   quarks        ();
-   timage        ();
-   zoomtf1       ();
-   zoomfit       ();
-   parallelcoord ();
-   clonepad      ();
-   hbars         ();
-   th2poly       ();
-   if (!gOptionR) {
-      std::cout << "**********************************************************************" <<std::endl;
-      if (!gTestsFailed) {
-         std::cout << "*  All the tests passed. :-)" <<std::endl;
-      } else {
-         if (gTestsFailed>1) std::cout << "*  " << gTestsFailed <<" tests failed. :-(" <<std::endl;
-         else                std::cout << "*  " << gTestsFailed <<" test failed. :-(" <<std::endl;
-      }
-      std::cout << "**********************************************************************" <<std::endl;
-
-      gBenchmark->Stop("stressGraphics");
-
-      //Print table with results
-      Bool_t UNIX = strcmp(gSystem->GetName(), "Unix") == 0;
-      if (UNIX) {
-         TString sp = gSystem->GetFromPipe("uname -a");
-         sp.Resize(60);
-         printf("*  SYS: %s\n",sp.Data());
-         if (strstr(gSystem->GetBuildNode(),"Darwin")) {
-            sp  = gSystem->GetFromPipe("sw_vers -productVersion");
-            sp += " Mac OS X ";
-            printf("*  SYS: %s\n",sp.Data());
-         }
-      } else {
-         const Char_t *os = gSystem->Getenv("OS");
-         if (!os) printf("*  SYS: Windows 95\n");
-         else     printf("*  SYS: %s %s \n",os,gSystem->Getenv("PROCESSOR_IDENTIFIER"));
-      }
-
-      printf("**********************************************************************\n");
-      printf("* ");
-      gBenchmark->Print("stressGraphics");
-
-      Double_t ct = gBenchmark->GetCpuTime("stressGraphics");
-      //normalize at 860 rootmarks on pcbrun4
-      const Double_t rootmarks = 860*(47.12/ct);
-
-      printf("**********************************************************************\n");
-      printf("*  ROOTMARKS =%6.1f   *  Root %-8s  %d/%d\n",rootmarks,gROOT->GetVersion(),
-             gROOT->GetVersionDate(),gROOT->GetVersionTime());
-      printf("**********************************************************************\n");
-   }
-}
+Int_t     gPS1RefNb[kMaxNumTests];
+Int_t     gPS1ErrNb[kMaxNumTests];
+Int_t     gPDFRefNb[kMaxNumTests];
+Int_t     gPDFErrNb[kMaxNumTests];
+Int_t     gJPGRefNb[kMaxNumTests];
+Int_t     gJPGErrNb[kMaxNumTests];
+Int_t     gPNGRefNb[kMaxNumTests];
+Int_t     gPNGErrNb[kMaxNumTests];
+Int_t     gPS2RefNb[kMaxNumTests];
+Int_t     gPS2ErrNb[kMaxNumTests];
+Bool_t    gWebMode = kFALSE;
+Bool_t    gOptionR = kFALSE;
+Bool_t    gOptionK = kFALSE;
+TH2F     *gH2 = nullptr;
+TFile    *gHsimple = nullptr;
+TFile    *gCernstaff = nullptr;
```

Comment:
If we're changing this old program might as well try to correct some of its mistakes, beginning from making all these raw pointers `std::unique_ptr` and removing all instances of `new` and `delete` from the file.